src/code42cli/logger/handlers.py

```diff
@@ -85,7 +85,7 @@ def handleError(self, record):
         them nowhere.
         """
         t, _, _ = sys.exc_info()
-        if t == BrokenPipeError:
+        if ConnectionError in t.__bases__:
```

Comment:
`issubclass(t, ConnectionError)` is the standard way to check this

Reply:
ha, thanks!

Reply:
I am going to push the changes, then pull it down to Windows and test there to make sure it works

Reply:
everything looks good, I have tested all cases on both mac and windows